apps/cnquery/cmd/builder/parse.go

```diff
@@ -612,14 +635,38 @@ func ParseTargetAsset(cmd *cobra.Command, args []string, providerType providers.
 		}
 	case providers.ProviderType_GOOGLE_WORKSPACE:
 		connection.Backend = providerType
+
+		// env vars have precedence over the --credentials-path arg
+		credsPath := readEnvs("GOOGLEWORKSPACE_CREDENTIALS", "GOOGLEWORKSPACE_CLOUD_KEYFILE_JSON", "GOOGLE_CREDENTIALS")
```

Comment:
These env vars seem to be used for terraform (at least that's what I saw thru a quick google search). Do we want to keep them? The google docs do not mention those

cc @chris-rock @imilchev

Reply:
we should only use env vars that either Google or terraform is using